packages/react/src/themes/teams/components/Popup/popupStyles.ts

```diff
@@ -6,11 +6,9 @@ const popupStyles: ComponentSlotStylesInput<PopupProps, PopupVariables> = {
   root: (): ICSSInJSStyle => ({}),
 
   popup: ({ variables }): ICSSInJSStyle => ({
-    zIndex: variables.zIndex,
     position: 'absolute',
     textAlign: 'left',
-    color: variables.contentColor,
-    background: variables.contentBackgroundColor,
```

Comment:
Moved to `popupContentStyles`...

Reply:
Talk to @kuzhelov. In #1121 he intentionally moved the styles from `popupContentStyles` to here.

Reply:
these styles should be left here, as we popup expects _any_ content to be rendered with this set of styles, not just `PopupContent`. Also, please, ensure that `PopupContent` just CSS-inherits this set of colors.

These two requirements are necessary for proper theming support